scripts/movePackageJson.ts

```diff
@@ -0,0 +1,11 @@
+import * as fs from 'fs';
```

Comment:
Hmm, no file comments in github :(

This file moves the package json to `dist/src` and prepares it for publishing (changes relative paths and deletes `private: true`)